app/src/main/java/com/example/gitrank/ui/Explore.kt

```diff
@@ -0,0 +1,35 @@
+package com.example.gitrank.ui
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.appcompat.app.AppCompatActivity
+import androidx.navigation.NavController
+import androidx.navigation.Navigation
+import com.example.gitrank.databinding.FragmentExploreBinding
+
+
+class Explore : Fragment() {
```

Comment:
I think it's best practise to name the classes well. For example, if we call this `Explore`, someone might not understand it well

We can rename it to `ExploreFragment`

Reply:
My bad, 
I will sure update them